js/build_map.js

```diff
@@ -9,16 +9,21 @@ var projection = albersUsaPr()
     .translate([chart_width / 2, chart_height / 2]);
 var buildPath = d3.geoPath()
     .projection(projection);
+    
+//Create container
+var container = d3.select('body')
```

Comment:
what does the container contain? could it be named to reflect the contents?

Reply:
Container contains everything.  The parent of all besides the footer and Header. I can rename it.